src/js/components/search/filters/awardAmount/SpecificAwardAmountItem.jsx

```diff
@@ -12,7 +12,17 @@ const propTypes = {
     searchSpecificRange: React.PropTypes.func
 };
 
+const ga = require('react-ga');
+
 export default class SpecificAwardAmountItem extends React.Component {
+    static logAmountRangeEvent(range) {
+        ga.event({
+            category: 'Search Page Filter Applied',
+            action: `Applied Award Amount Range Filter`,
```

Comment:
So the tick marks are an ES6 feature called [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) which are used to inject dynamic content (variables) inside an otherwise static string. In most cases where you're just using a string, you should use the single quotes (in this case, `action`).